modules/ibi-generate-seed-image.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+// * edge_computing/ztp-image-based-upgrade.adoc
+// * edge_computing/factory-image-based-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` CR. The Operator checks for required system configurations, performs any necessary system cleanup before generating the seed image, and launches the image generation. The seed image generation includes the following tasks:
+
+* Stopping cluster operators
+* Preparing the seed image configuration
+* Generating and pushing the seed image to the image repository specified in the `SeedGenerator` CR
+* Restoring cluster operators
+* Expiring seed cluster certificates
+* Generating new certificates for the seed cluster
+* Restoring and updates the `SeedGenerator` CR on the seed cluster
+
+[NOTE]
+====
+The generated seed image does not include any site-specific data.
+
+To avoid including any {rh-rhacm-first} resources in your seed image, you need to disable all optional {rh-rhacm} add-ons for seed image generation.
+====
+
+.Prerequisites
+
+* You have deployed a managed a {sno} cluster with a DU profile.
+* Install the {lcao} in the hub cluster.
+* Log in as a user with `cluster-admin` privileges.
+* Review the required versions of software components in the section _Software prerequisites for the image-based installation_.
+* You created an an extra partition in the seed cluster. See _Configuring a shared container directory between `ostree` stateroots_ for further information.
+
+.Procedure
+
+. Detach the cluster to delete any cluster-specific resources from the seed cluster that must not be in the seed image:
+
+.. Manually detach the seed cluster by running the following command:
++
+[source,terminal]
+----
+$ oc delete managedcluster sno-worker-example
+----
+
+... Wait until the `ManagedCluster` CR is removed. Once the CR is removed, create the proper `SeedGenerator` CR. The {lcao} cleans up the {rh-rhacm} artifacts.
+
+. Create the `Secret`.
+
+.. Create the authentication file by running the following command:
++
+--
+.Authentication file
+[source,terminal]
+----
+$ MY_USER=myuserid
+$ AUTHFILE=/tmp/my-auth.json
+$ podman login --authfile ${AUTHFILE} -u ${MY_USER} quay.io/${MY_USER}
+----
+
+[source,terminal]
+----
+$ base64 -w 0 ${AUTHFILE} ; echo
+----
+--
+
+.. Copy the output into the `seedAuth` field in the `Secret` YAML file named `seedgen` in the `openshift-lifecycle-agent` namespace.
```

Comment:
This sentence needs re-phrasing.
Looking at the example below, the file name  should be secretseedgenerator.yaml
The yaml file isn't in any namespace.

Reply:
Updated to say ` ...in a `Secret` CR named `seedgen` ...` so we don't get confused with filenames